istioctl/cmd/root.go

```diff
@@ -313,3 +356,14 @@ func unsupportedCmd(cmd *cobra.Command, alternative string) *cobra.Command {
 		},
 	}
 }
+
+// disableUnsupportedCmds is used to disable all the commands that are not supported.
```

Comment:
One thing I wonder - is it possible that we would not support a single sub-command among others? IIUC this only works on the top-level command rights. If the aforementioned case would be needed, we might want to rework this part to be recursive:

```golang
func VisitAll(command *cobra.Command, apply func(command *cobra.Command)) {
	for _, child := range command.Commands() {
		VisitAll(child, apply)
	}
	apply(command)
}


// actual call

VisitAll(root, func(c *cobra.Command) {
	if unsupportedCmds[c.Name()] {
		parent := c.Parent()
		parent.Remove(c)
		parent.AddCommand(unsupportedCmd(c, UnsupportedCmdsMsg, buildUnsupportedCmdsAlternativeMsg(UnsupportedCmdsAlternativeMsg)[c.Name()]))
})

```

Reply:
I'm not sure this case could happen. It would also imply to get another format of the build flag, like `cmd.disabledCmds=cmd1.sub-cmd,cmd2`
@dgn WDYT?